app/src/store/StoreAlerts.re

```diff
@@ -0,0 +1,30 @@
+open Molecule.Alert;
```

Comment:
The reason why we have to import this is for severity only, right? For how I understand the dependency graph the store shouldn't know about the high-up layers (anything visual here).